lib/App/DuckPAN.pm

```diff
@@ -207,6 +207,12 @@ sub get_reply {
 	return $return;
 }
 
+sub ask_yn {
```

Comment:
Can we not just use [get_reply](https://github.com/srvsh/p5-app-duckpan/blob/generic_templates/lib/App/DuckPAN.pm#L203)?